pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -162,6 +163,11 @@ func (h *NetworkUtilsHandler) ConfigureIpv6Forwarding() error {
 	return err
 }
 
+func (h *NetworkUtilsHandler) IsIpv4Primary() bool {
+	podIp := os.Getenv("MY_POD_IP")
```

Comment:
This should panic in case we don't find the variable. If somebody removes it from the spec, we should provide a clear unrecoverable panic message.